src/Main.java

```diff
@@ -8,11 +8,31 @@ public static void main(String[] args) {
         Integer num1;
         Integer num2;
         String operation;
-//        Integer plusSolution;
-//        Integer minusSolution;
-//        Integer timesSolution;
-//        Integer dividedSolution;
         Integer solution;
+       /* Integer one;
+        Integer two;
```

Comment:
I see where you are trying to go with this. But for now, only use ```Integer``` objects for the two input numbers and one for the solution. 

The steps should be:
1. Read in first number as a ```String```. This will be something like "one" or "two." Simplest way is with ```if-else``` statements. This will look like: ```if(input1.equalsIgnoreCase("one")) { number1 = 1; }```
2. Convert that ```String``` into and ```Integer```. This is something you could Google ("Convert String to Integer Java")
3. Read operator as a ```String```. Do not need to do anything with it yet.
4. Repeat steps (1) and (2) to read the second number.
5. Use ```if-else``` statements to add, subtract, multiply, or divide based one the operator. Should be something like ```if(operator.equalsIgnoreCase("plus") { solution = number1 + number2; }```
6. Output the solution. Something like ```System.out.printLn(number1 + operator + number2 + "= " + solution);```

We can worry about breaking it down into multiple methods and error conditions and everything else later on!